src/Copy/BrandCopy.elm

```diff
@@ -606,6 +606,52 @@ brandCopy key =
         StoryThree4Message ->
             CopyText ""
 
+        StoryFourTitle ->
+            CopyText "Lucille"
+
+        StoryFourTeaser ->
+            CopyText "Lucille had never needed to complain about her GP surgery, but when her new medication was messed about she had to speak up."
+
+        StoryFourTeaserImageAlt ->
+            CopyText "✨"
+
+        StoryFour1Quote ->
+            CopyText """My surgery is in a little village, I’ve been with them for 9 years and had never had to make a complaint or get cross with them. But I won’t let the status quo stand when things are unfair. So when I was messed around getting the new medication I needed to go about my life normally I did complain."""
```

Comment:
Either remove the `"""` or move these to multi-line.

Reply:
only from these stories or from all the others? I guess if we want to be consistent, we might want to remove the eliminate the triple quotes from all...

Reply:
Other way round. I prefer the multi-line. It's much easier to read/ check. The other brands do use that.

Reply:
Since we had some new developers on this project, we didn't always enforce that consistency.